web_tree_image_tooltip/readme/ROADMAP.rst

```diff
@@ -0,0 +1,2 @@
+Idea: make easy to use small images in tree view to save bytes in network
```

Comment:
This is already supported AFAIK in standard Odoo. That's why the module was renamed to tooltip.